common/scala/src/main/scala/whisk/core/entity/WhiskAction.scala

```diff
@@ -88,6 +88,8 @@ abstract class WhiskActionLike(override val name: EntityName) extends WhiskEntit
       parameters.definedParameters
     } else Set.empty[String]
 
+  def entityType = "action"
```

Comment:
Could be a val.

Reply:
Ack. Made it an instance val. Used def earlier to avoid extra memory usage for in memory representation of entity. However val looks like the right way now